Upstream: No (not needed)
Source: Daniel Mierswa <impulze@impulze.org>
Reason: the dtags patch in the exlib makes this excessive test fail

diff -Naur binutils-2.21.51.orig/ld/testsuite/ld-x86-64/x86-64.exp binutils-2.21.51/ld/testsuite/ld-x86-64/x86-64.exp
--- binutils-2.21.51.orig/ld/testsuite/ld-x86-64/x86-64.exp	2011-03-12 15:38:31.000000000 +0100
+++ binutils-2.21.51/ld/testsuite/ld-x86-64/x86-64.exp	2011-04-09 12:40:36.000000000 +0200
@@ -194,7 +194,6 @@
 run_dump_test "ilp32-1"
 run_dump_test "ilp32-2"
 run_dump_test "ilp32-3"
-run_dump_test "ilp32-4"
 run_dump_test "ilp32-5"
 run_dump_test "ia32-1"
 run_dump_test "ia32-2"
